### ISSUE TITLE
Deprecate tertiary text color

### ISSUE THREAD

**Issue by jouni** [vaadin/vaadin-lumo-styles#98]
The current tertiary text color has a contrast of 2.92 :1 on the base color (white), which does not meet any accessibility requirements. The usefulness of this color property is questionable.

The only place I remember it being used at the moment is in date picker, for the week day names and week numbers. That’s a poor design decision on my part.

I propose that we deprecate the tertiary text color, and remove it in the next major version of Lumo.

Technically, we could map the property by default to the secondary text color, to improve accessibility. Not sure if there’s a way to give any deprecation warning about this, if some custom CSS is using it?

**Comment by web-padawan** [comment]
Added this to the list of breaking changes. I don't think there is a way to give a warning without using a tool like Stylelint.

**Comment by jouni** [comment]
Well, we can at least mark it deprecated in the documentation and in code, to discourage its use.
